chef/cookbooks/provisioner/recipes/base.rb

```diff
@@ -350,7 +350,8 @@
 crowbar_node = node_search_with_cache("roles:crowbar").first
 address = crowbar_node["crowbar"]["network"]["admin"]["address"]
 protocol = crowbar_node["crowbar"]["apache"]["ssl"] ? "https" : "http"
-server = "#{protocol}://#{address}"
+server = "#{protocol}://#{NetworkHelper.wrap_ip(address)}"
+password = crowbar_node["crowbar"]["users"]["crowbar"]["password"]
```

Comment:
Where did this line come from? This is the cause of the current CI failure, and on current code, is roughly replicated and replaced a few lines down. Can you remove this, rebase and push it up?